src/index.html

```diff
@@ -7,8 +7,74 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin=" "
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home" class="logo">
+        <img
+          src="./images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="navList">
+          <li class="link">
+            <a href="#" class="navLink is-active">
```

Comment:
```suggestion
        <ul class="nav-list">
          <li class="nav-item">
            <a href="#" class="nav-link is-active">
```
name classes in kebab-case or use BEM notation `nav nav__list nav__link nav__item`